### PR TITLE
blank slate documentation

### PR DESCRIPTION
**Pull request by craigcwaveformlabs**
first attempt at adding blank slate documentation.

### CODE REVIEW

File: docs/components/blank-slate.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+  <div class="Grid-row">
+    <div class="Grid-col sm-12 md-6 lg-6">
+      <div class="UsageBlock UsageBlockCorrect">
+        <div class="UsageBlock-body">
+          <div class="UsageBlock-body--max-width">
+            <h5>Do</h5>
+            <ul>
+              <li>When introducing and articulating the benefit of using such a feature</li>
+              <li>When explaining a possible complicated feature by focusing on just a few benefits</li>
+              <li>When providing an onboarding experience to new users</li>
```
Comment (atikamary) [inline docs/components/blank-slate.md]: Maybe mention something about making complicated features easy to understand

File: docs/components/blank-slate.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+<div class="u-padding-top--large u-padding-bottom--large">
+  <div class="Grid-row">
+    <div class="Grid-col sm-12 md-6 lg-6">
+      <div class="UsageBlock UsageBlockCorrect">
+        <div class="UsageBlock-body">
+          <div class="UsageBlock-body--max-width">
+            <h5>Do</h5>
+            <ul>
+              <li>Use no more than one primary call to action</li>
```
Comment (atikamary) [inline docs/components/blank-slate.md]: I think this rule is currently broken in a couple places, banking comes to mind

Comment (craigcwaveformlabs) [reply docs/components/blank-slate.md]: Yeah good point. Banking is one such place

File: docs/components/blank-slate.md (hunk 1 above)
Comment (craigcwaveformlabs) [reply docs/components/blank-slate.md]: 👍

**Pull request merged**